ArduSub/GCS_Sub.h

```diff
@@ -27,6 +27,16 @@ class GCS_Sub : public GCS
 
     bool vehicle_initialised() const override;
 
+protected:
+
+    // maximum amount of time (in microseconds) that must remain in
```

Comment:
Same.

Reply:
Fixed.  It was even there in the function name!